apps/wallet/README.md

```diff
@@ -59,3 +59,18 @@ After building the app, the extension needs to be installed to Chrome. Follow th
 ```
 pnpm wallet test
 ```
+
+## Run Wallet e2e tests
+
+Wallet e2e tests are running with lower epoch duration
+Run this in 1 terminal:
+
+```
+cargo run --bin sui-test-validator -- --epoch-duration-ms 10000
```

Comment:
I think we can skip this. We can update the [playwright.config.ts](https://github.com/MystenLabs/sui/blob/main/apps/wallet/playwright.config.ts#L60) to start the network with the required params. And playwright will start the service automatically if not running.

So I think in the end just running `pnpm wallet playwright test` would be enough.

(We could also add a note to stop any other instances of sui network)